### ISSUE TITLE
bug or feature ?

### ISSUE THREAD

**Issue by neutrinoceros** [PyCQA/isort#1631]
I'm curious if this behaviour is intended where isort doesn't reorder anything but combines comments attached to lines where I import the same module under two different names

```patch
isort --diff isort_comments.py

--- path/to/cwd/isort_comments.py:before 2021-01-03 18:20:37.072373
+++ path/to/cwd/isort_comments.py:after	2021-01-03 18:22:04.439485
@@ -1,2 +1,2 @@
-import a # a comment
-import a as b # b comment
+import a  # a comment; b comment
+import a as b  # a comment; b comment
```

**Comment by timothycrosley** [comment]
This looks like a bug to me! Thanks for reporting!

**Comment by timothycrosley** [comment]
This is fixed in develop and will be released in the 5.8.0 release of isort.

Thanks!

~Timothy
